src/main/proto/dex.proto

```diff
@@ -170,4 +204,89 @@ message SubmitProposal {
     bytes proposer = 4;
     repeated Token initial_deposit = 5;
     int64  voting_period = 6;
-}
\ No newline at end of file
+}
+
+message Deposit {
+    // 0xA18A56E5
+  int64 proposal_id = 1;
+  bytes   depositer = 2;
+  repeated Token amount = 3;
+}
+
+message Description {
+   string moniker  = 1;
+   string identity = 2;
+   string website = 3;
+   string details = 4;
+}
+
+message Commission {
+    int64 rate = 1;
+    int64 max_rate = 2;
+    int64 max_change_rate = 3;
+}
+
+message CreateValidator  {
+    // 0xEB361D01
+  Description description = 1;
+  Commission    commission = 2;
+  bytes delegator_address  = 3;
+  bytes validator_address  = 4;
+  bytes  pubkey = 5;
+  Token      delegation = 6;
+}
+
+message RealCreateValidator {
```

Comment:
`RealCreateValidator ` is not consistent with bnbchain, please keep consistent with chain defenition.